.github/workflows/ci.yaml

```diff
@@ -2,6 +2,7 @@ on:
   push:
     branches:
       - master
+      - angular-skeleton
```

Comment:
I don't think we'll need to add an additional branch, we should just confirm that this works with a draft PR.